src/openshift/cluster.ts

```diff
@@ -120,19 +124,50 @@ export class Cluster extends OpenShiftItem {
     }
 
     static async getUrl(): Promise<string | null> {
-        const k8sConfig = new KubeConfigUtils();
         const clusterURl = await Cluster.getUrlFromClipboard();
-        const createUrl: QuickPickItem = { label: '$(plus) Provide new URL...'};
-        const clusterItems = k8sConfig.getServers();
-        const choice = await window.showQuickPick([createUrl, ...clusterItems], {placeHolder: 'Provide Cluster URL to connect', ignoreFocusOut: true});
-        if (!choice) return null;
-        return (choice.label === createUrl.label) ?
-            window.showInputBox({
-                value: clusterURl,
-                ignoreFocusOut: true,
-                prompt: 'Provide new Cluster URL to connect',
-                validateInput: (value: string) => Cluster.validateUrl('Invalid URL provided', value)
-            }) : choice.label;
+        return await Cluster.showQuikPick(clusterURl);
+    }
+
+    private static async showQuikPick(clusterURl: string): Promise<string> {
+        return await new Promise<string | null>((resolve, reject) => {
+            const k8sConfig = new KubeConfigUtils();
+            const addBtn = new quickBtn(new ThemeIcon('plus'), 'Add');
+            const switchBtn = new quickBtn(new ThemeIcon('arrow-swap'), 'Switch');
+            const deleteBtn = new quickBtn(new ThemeIcon('trash'), 'Delete');
+            const createUrl: QuickPickItem = { label: 'Provide new URL...', buttons: [addBtn] };
```

Comment:
**addBtn** is not needed, just recognize Add quickpick in **onDidAccept** listener.